File: dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/TaskModules/TaskModulesOptions.cs
namespace Microsoft.Teams.AI
{
    /// <summary>
    /// Options for TaskModules class.
    /// </summary>
    public class TaskModulesOptions
    {
        /// <summary>
        /// Data field to use to identify the verb of the handler to trigger.
        /// </summary>
        /// <remarks>
        ///  When a task module is triggered, the field name specified here will be used to determine
        /// the name of the verb for the handler to route the request to.
        /// Defaults to a value of "verb".
        /// </remarks>
        public string? TaskDataFilter { get; set; }
    }
}

File: python/packages/ai/teams/task_modules/task_modules_options.py
"""
Copyright (c) Microsoft Corporation. All rights reserved.
Licensed under the MIT License.
"""

from __future__ import annotations

from dataclasses import dataclass


@dataclass
class TaskModulesOptions:
    task_data_filter: str = "verb"
    """
    Data field to use to identify the verb of the handler to trigger.
    When a task module is triggered, the field name specified here will be used to determine
    the name of the verb for the handler to route the request to.
    """
